styles/blog.css

```diff
@@ -0,0 +1,188 @@
+* {
+   box-sizing: border-box;
+   font-family: Arial, Helvetica, sans-serif
+}
+
+/* header setting */
+header {
+  background-image: url(../images/header-image.jpg);
+  padding-left: 10%;
+  padding-top: 5%;
+  padding-right: 10%;
+  padding-bottom: 15%;
+}
+
+h1 {
+  color: #81B741;
+  font-family: "Lucida Handwriting", "Brush Script M7", cursive;
+  font-size: 150%;
+}
+
+.shopping-cart{
+  width: 1.5rem;
+  text-align: right;
+}
+
+.check-out {
+  background-color: #81B741;
+  color: white;
+  padding: 10px 4px;
+  text-align: center;
+}
+
+.headernavigation {
+    float: left;
+    display: block;
+    text-align: left;
+    align-content: center;
+    list-style-type: none;
+    overflow: hidden;
+    color: #083B60;
+    padding-left: 15%;
+    padding-top: 1.5%;
+    padding-bottom: 1.5%;
+    background: lightgrey;
+    font-weight: bold;
+  }
+
+.headernavigation a {
+    color: #083B60;
+    text-decoration: none;
+    padding: 5px;
+  }
+
+/* main text setting */
+h2 {
+  color: #083B60;
+  font-family: Garamond, Georgia, Serif;
+  padding: 10% 15% 0% 15%;
+}
+
+h3 {
+  color: #083B60;
+  font-family: Garamond, Georgia, Serif;
+}
+
+.blogitem {
+  padding: 10%;
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.blogimage {
+  height: 100%;
+  width: 100%;
+}
+
+
+/* footer setting */
+footer {
+  background-color: #083B60;
+  background-position: bottom;
+  background-size: cover;
+  position: static;
+  color: white;
+  font-size: 17px;
+  padding: 20px 0px;
+  text-align: center;
+
+}
+
+.footerImage {
+  width: 20rem;
+  height: auto;
+}
+
+.footernavigation {
+    font-size: 20px;
+    list-style-type: none;
+    overflow: hidden;
+    color: white;
+  }
+
+.footernavigation a {
+    color: white;
+    text-decoration: none;
+    padding: 5px;
+  }
+
+/* grid layout setting */
+.col {
+   float: left;
```

Comment:
Yeah. Don't use float for grids. Find a more modern approach using flexbox or css grid

Reply:
I tried to avoid any grid layout but had to put it back to make the responsive design work, and with the grid I couldn't figure out how to avoid floating them for the layout.  I will be sure to think about this more and see if I can find a solution to this!